CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 9.1.0
+  - Set number_of_shards to 1 and document_type to '_doc' for es 7.x clusters #741 #747
```

Comment:
I think these changes will need to be listed as breaking changes in the 7.0 docs. Is there a way to track that or mark them as breaking? Perhaps a simple "Breaking: " in front of the line?

Reply:
Btw, do the plugins follow semver?

Reply:
They do follow semver. Here I made the call to not call breaking as it changes only the template for 7.x and 7.0 isn't GA yet, what do you think?

Reply:
Ah, I see. It's going to be (arguably) a breaking change when we release Logstash 7.0, but I agree it's not yet a breaking change. Not sure what to do, I'd just like to highlight this in the breaking docs once we release Logstash 7.0.

Reply:
We typically have the migration/upgrading docs, and this definitely should be noted there, and perhaps have a note in the breaking changes doc about it as well.
Another argument for not calling it a breaking change at logstash level is that we'll likely ship this plugin change in logstash 6.x. since at plugin level we didn't break bwc.